docs/source/reference/numpysupported.rst

```diff
@@ -338,6 +338,7 @@ The following top-level functions are supported:
 * :func:`numpy.array` (only the 2 first arguments)
 * :func:`numpy.array_equal`
 * :func:`numpy.asarray` (only the 2 first arguments)
+* :func:`numpy.asarray_chkfinite` (only the first first arguments)
```

Comment:
```suggestion
* :func:`numpy.asarray_chkfinite` (only the 2 first arguments)
```

Reply:
Yeah, either that or `only the first two arguments`, but I guess that would be inconsistent with the rest of the doc?